projects/storefrontapp-e2e-cypress/cypress/integration/mobile/register-mobile.e2e-spec.ts

```diff
@@ -13,8 +14,7 @@ describe(`${formats.mobile.width + 1}p resolution - Register`, () => {
     cy.viewport(formats.mobile.width, formats.mobile.height);
   });
 
-  // Behavior changed to automatic login.
-  it('should login when trying to register with the same email and correct password', () => {
+  it('should register and redirect to login page', () => {
```

Comment:
This test could be simplified only to this (extra checks no longer have value):
```
    waitForHomePage();
    register.registerUser(user);
    register.verifyGlobalMessageAfterRegistration();
    register.navigateToTermsAndConditions();
    register.checkTermsAndConditions();
```

Reply:
Not sure what has t&c page to do here?